app/src/main/java/com/li/musicmp3/data/source/local/ultil/LoadMusicTask.kt

```diff
@@ -0,0 +1,26 @@
+package com.li.musicmp3.data.source.local.ultil
+
+import android.os.AsyncTask
+import com.li.musicmp3.R
+import com.li.musicmp3.data.model.Song
+import com.li.musicmp3.data.source.local.OnLoadMusicCallBack
+import java.lang.Exception
+
+class LoadMusicTask(
+    private val callback: OnLoadMusicCallBack
+    , val loadSong: () -> List<Song>
```

Comment:
Xem thứ tự param trong conventions nhé 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md
```suggestion
  val loadSong: () -> List<Song>,
    private val callback: OnLoadMusicCallBack
  
```